iogt/templates/base.html

```diff
@@ -166,12 +174,14 @@
               <img src="{% static 'img/footer_login_icon.png' %}" width="30" class="menu-list__item--icon" />{% trans "Profile" %}</a>
             {% endif %}
           </li>
+          {% if languages|length > 1 %}
```

Comment:
Here you can just write it as `{% if languages %}` 

See: http://stackoverflow.com/questions/902034/how-can-i-check-the-size-of-a-collection-with-django-templates